Simcoe/Simcoe.swift

```diff
@@ -326,6 +326,39 @@ public final class Simcoe {
         }
```

Comment:
I think you may have quoted too much above in [line 271](https://github.com/prolificinteractive/simcoe/pull/19/files#diff-4390d00a82dcc4b10a4d8a3132b44fd0R271), all the text is blue